rules/falco_rules.yaml

```diff
@@ -2255,11 +2255,19 @@
 - macro: network_tool_procs
   condition: (proc.name in (network_tool_binaries))
 
+- macro: user_known_package_manager_in_container_conditions
```

Comment:
Can you rename the macro to `user_known_package_manager_in_container`, the name is too long. And I couldn't think of any better :(

Reply:
Fixed!